src/Sdk/DTLogging/Logging/ValueEncoders.cs

```diff
@@ -80,6 +81,24 @@ public static String TrimDoubleQuotes(String value)
             return trimmed;
         }
 
+        public static String RemovePowerShellSpecialCharacters(String value)
+        {
+            var trimmed = string.Empty;
+            if (!string.IsNullOrEmpty(value))
+            {
+                // Match any number of "+&;" characters and one extra character
+                Regex pwshSpecialChars = new Regex(@"^[+&;]*.", RegexOptions.Compiled | RegexOptions.CultureInvariant);
```

Comment:
```
^[+&;]+.
```
I think you want a plus instead of a *

Reply:
Might be good to consult a guide on special characters for powershell and make sure to test them all in relation to &